src/protocol/Makefile.am

```diff
@@ -13,6 +13,8 @@ GOLDILOCKS_CPPFLAGS = \
 AM_CPPFLAGS = -I$(top_srcdir)/include -I$(top_srcdir)/src $(GOLDILOCKS_CPPFLAGS)
 AM_CFLAGS = @WARNING_FLAGS@
 
+#AM_CFLAGS += -I$(BOLOS_ENCLAVE_SDK)/api
```

Comment:
This commented line could be removed (or if one wants to keep it for a serious reason, then add a comment explaining why)

Reply:
Removed